$MirOS$

	hunk #1: MirLibtool is libtool 1.5 based

	hunk #2: Wee Tee Eff?

--- configure.ac.orig	Mon Jan 30 03:44:19 2006
+++ configure.ac	Thu Mar 27 17:10:07 2008
@@ -7,8 +7,9 @@ AM_MAINTAINER_MODE
 AC_CONFIG_HEADERS([config.h])
 AC_PROG_MAKE_SET
 AM_PROG_LIBTOOL
-LT_INIT
-LT_LANG(C)
+dnl comment these out: fakeroot uses GNU libtool 2.1a originally
+dnl LT_INIT
+dnl LT_LANG(C)
 
 AC_ARG_WITH([ipc],
 	    AS_HELP_STRING([--with-ipc@<:@=IPCTYPE@:>@],
@@ -330,8 +331,16 @@ case $target_cpu:$target_os in
 	(*:netbsd*)
 		libcpath="/usr/lib/libc.so.12"
 		;;
-	(*:openbsd*)
-		libcpath="/usr/lib/libc.so.25.2"
+	(*:openbsd*|*:mirbsd*)
+		libcpath=$(/bin/ls -1 /usr/lib/libc.so.* | \
+		    sort -nt. -k3 | tail -1)
+		;;
+	(*:midnightbsd*)
+		libcpath=$(/bin/ls -1 /lib/libc.so.* | \
+		    sort -nt. -k3 | tail -1)
+		;;
+	(*:darwin*)
+		libcpath="/usr/lib/libSystem.dylib"
 		;;
 	(*:hpux*)
 		libcpath="/usr/lib/hpux32/libc.so.1"
